src/components/backend/utils.js

```diff
@@ -0,0 +1,12 @@
+export const downloadData = (filename, data) => {
```

Comment:
😂 is this the right way to do this...... impressive but feels wrong haha

Reply:
I got this from chatgpt... Could you suggest a better way?

Reply:
Adding a TODO and closing this comment.